pdc/apps/component/tests.py

```diff
@@ -379,7 +379,6 @@ def test_dist_git_web_url_without_inherited_dist_git_branch(self):
 class ReleaseComponentRESTTestCase(TestCaseWithChangeSetMixin, APITestCase):
 
     fixtures = [
-        "pdc/apps/release/fixtures/release_type.json",
```

Comment:
If this is not related to the upgrading codes, if we need to remove this from this patch?

Reply:
True, it didn't have to be here, but it's at least separate commit "Remove reference to non-existing test fixture".

Reply:
OK, in separate commit makes sense.